source/module_base/test/tool_check_test.cpp

```diff
@@ -0,0 +1,98 @@
+#include "../tool_check.h"
+#include "gtest/gtest.h"
+#include <fstream>
+#include <cstdio>
+
+/************************************************
+*  unit test of class ToolCheckTest
+***********************************************/
+
+/**
+ * - Tested Function
+ *   - ModuleBase::CHECK_NAME
+ *	- check the next input from ifs is std::string
+ *
+ *   - ModuleBase::CHECK_INT
+ *	- check the next input from ifs is int
+ *
+ *   - ModuleBase::CHECK_DOUBLE
+ *	- check the next input from ifs is double
+ *
+ *   - ModuleBase::CHECK_STRING
+ *	- check the next input from ifs is string
+ *
+ */
+
+
+class ToolCheckTest : public testing::Test
+{
+    protected:
+        std::ofstream ofs;
```

Comment:
You have inconsistent use of tab and space for indentation. Please reformat your code.

Reply:
done